docs/UserGuide.md

```diff
@@ -515,17 +515,48 @@ Format: `undone-o INDEX`
 **Examples:**
 * `undone-o 1` Marks the order at index 1 in the order list currently shown as undone.
 
-### 4.6 Exiting the program: `exit`
+### 4.6 Recipe
```

Comment:
Do you think we should move this before order so that by the time the user reads the order section, he/she will be aware of what a recipe means in BTBB?

Reply:
Currently is by alphabetical order. But what you said also make sense. @zognin thoughts?